tests/nexus_api_test.go

```diff
@@ -184,15 +190,22 @@ func (s *ClientFunctionalSuite) TestNexusStartOperation_Outcomes() {
 
 		go s.nexusTaskPoller(ctx, tc.incomingService.Spec.TaskQueue, tc.handler)
 
+		eventuallyTick := 500 * time.Millisecond
```

Comment:
Why does the test type have to change the tick speed?
Are you trying to prevent concurrent requests?

Reply:
I'm trying to prevent the Eventually block from rerunning while the handler is waiting for the timeout because that request will fail since we use a poller that only handles one task.